yarn.lock

```diff
@@ -7839,6 +7839,11 @@ cypress-plugin-tab@1.0.5:
   dependencies:
     ally.js "^1.4.1"
 
+cypress-real-events@1.7.6:
+  version "1.7.6"
+  resolved "https://eurostar.jfrog.io/eurostar/api/npm/npm/cypress-real-events/-/cypress-real-events-1.7.6.tgz#6f17e0b2ceea1d6dc60f6737d8f84cc517bbbb4c"
```

Comment:
👀 🚆 :)

Setting scopes in your root `.npmrc` might help:
https://docs.npmjs.com/cli/v9/using-npm/scope#associating-a-scope-with-a-registry

Reply:
What, this isn't the train place?

Thanks @arelra [fixed now](https://github.com/guardian/dotcom-rendering/pull/7129/commits/2467f9e5bda9aa2d72331dbec3ffdf7e1cf150bb). I should have setup scopes a long time ago 😊